geniza/annotations/views.py

```diff
@@ -125,9 +135,9 @@ def delete(self, request, *args, **kwargs):
         # should use etag / if-match
         # deleted uuid should not be reused (relying on low likelihood of uuid collision)
         anno = self.get_object()
-        anno.delete()
-
-        # create log entry to document deletion
+        # create log entry to document deletion *BEFORE* deleting
```

Comment:
Good call!

Reply:
figured it out when I wrote the tests! it doesn't work otherwise (for obvious reasons)